src/server/server_family.cc

```diff
@@ -1253,10 +1253,13 @@ void ServerFamily::UpdateMemoryGlobalStats() {
   double rss_oom_deny_ratio = ServerState::tlocal()->rss_oom_deny_ratio;
   if (rss_oom_deny_ratio > 0) {
     size_t memory_limit = max_memory_limit.load(memory_order_relaxed) * rss_oom_deny_ratio;
-    if (total_rss > memory_limit && accepting_connections_ && HasPrivilegedInterface())
+    if (total_rss > memory_limit && accepting_connections_ && HasPrivilegedInterface()) {
+      LOG_EVERY_N(WARNING, 1000, "Accepting connections stopped because used memory is over limit");
```

Comment:
we use `LOG_EVERY_T(WARNING, ...)` for this as it's a more controllable behavior

Reply:
changed to every 10 seconds